api/types/error_response.go

```diff
@@ -7,7 +7,10 @@ package types
 // swagger:model ErrorResponse
 type ErrorResponse struct {
 
-	// The error message.
+	// The error message. It is formatted if multiple error messages are coalesced.
 	// Required: true
 	Message string `json:"message"`
+
+	// The list of unformatted error messages.
+	RawMessages []string `json:"raw_messages"`
```

Comment:
Do we anticipate we'd need more structured information at some point? Thinking of the structure that the OCI distribution spec chose; https://github.com/opencontainers/distribution-spec/blob/main/spec.md#error-codes

/cc @corhere @tianon 


Maybe we don't need it, but "now" is the moment to choose :)

Reply:
Discussing in the sync; and we can use a struct (currently with just one field); could be `message` or `detail` (see OCI spec)

Reply:
`message` probably is most appropriate for our use 👍